hw_sys_os/rm.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+import os
+import sys
+import shutil
+import argparse
+description = ''' ./rm.py [OPTION]... [FILE]...
+Remove (unlink) the FILE(s).\n'''
+parser = argparse.ArgumentParser(description)
+parser.add_argument('-r', '-R', '--recursive', action='store_true', default=False, required=False,  help='remove directories and their contents recursively')
+parser.add_argument('input', nargs='*')
+args = parser.parse_args()
```

Comment:
Лучше делать отступ хотя бы в одну строку от блока с аргументами для лучшей читаемости
```suggestion
parser.add_argument('input', nargs='*')
args = parser.parse_args()

```